model/src/main/kotlin/config/ScanStorageConfiguration.kt

```diff
@@ -72,9 +72,9 @@ data class PostgresStorageConfiguration(
     val url: String,
 
     /**
-     * The name of the database to use.
+     * The schema used by the database. Default is public.
```

Comment:
I'd say `Defaults to the "public" schema which exists by default.`

Reply:
In the commit message:

> Postgres define the

"defines"

> as not usual create custom schema on test databases.

I do not get this sentence. Is there an "it is" missing after "as"? Also "*a* custom schema". And why "test" database?